src/views/TechnicalAssistance/TrbAdminTeamHome.tsx

```diff
@@ -0,0 +1,648 @@
+
+    return [
+      submissionDate,
+      r.name,
+      requester,
+      i18next.t<string>(`technicalAssistance:table.requestTypes.${r.type}`),
+      trbLead,
+      i18next.t<string>(`technicalAssistance:table.requestStatus.${r.status}`),
+      trbConsultDate
+    ];
+  });
+
+  return [trbRequestsCsvHeader, ...rows];
+}
+
+function CsvDownloadLink({
+  children,
+  data,
+  filename
+}: {
+  children?: React.ReactNode;
+  data: any;
+  filename?: string;
+}) {
+  return (
+    <CSVLink className="usa-link" data={data} filename={filename}>
+      <IconFileDownload className="text-middle margin-right-1" />
+      {children}
+    </CSVLink>
+  );
+}
+
+function SubmissionDateCell({
+  value
+}: CellProps<
+  TrbAdminTeamHomeRequest,
+  TrbAdminTeamHomeRequest['form']['submittedAt']
+>) {
+  return value ? formatDateLocal(value, 'MM/dd/yyyy') : '';
+}
+
+function RequestNameCell({
+  value,
+  row
+}: CellProps<TrbAdminTeamHomeRequest, TrbAdminTeamHomeRequest['name']>) {
+  return (
+    <UswdsReactLink to={`/trb/task-list/${row.original.id}`}>
+      {value}
+    </UswdsReactLink>
+  );
+}
+
+function RequesterCell({ row }: CellProps<TrbAdminTeamHomeRequest>) {
+  return getPersonVal(
+    row.original.requesterInfo.commonName,
+    row.original.requesterComponent
+  );
+}
+
+function TrbLeadCell({ row }: CellProps<TrbAdminTeamHomeRequest>) {
+  return getPersonVal(
+    row.original.trbLeadInfo.commonName,
+    row.original.trbLeadComponent
+  );
+}
+
+type TrbRequestsTableProps = {
+  requests: TrbAdminTeamHomeRequest[];
+};
+
+function TrbNewRequestsTable({ requests }: TrbRequestsTableProps) {
+  const { t } = useTranslation('technicalAssistance');
+
+  // @ts-ignore
+  const columns = useMemo<Column<TrbAdminTeamHomeRequest>[]>(() => {
+    return [
+      {
+        Header: t<string>('table.header.submissionDate'),
+        accessor: 'form.submittedAt',
+        Cell: SubmissionDateCell
+      },
+      {
+        Header: t<string>('table.header.requestName'),
+        accessor: 'name',
+        Cell: RequestNameCell
+      },
+      {
+        Header: t<string>('adminHome.requester'),
+        accessor: 'requesterInfo.commonName',
+        Cell: RequesterCell
+      },
+      {
+        Header: t<string>('adminHome.requestType'),
+        accessor: ({ type }) => t(`table.requestTypes.${type}`)
+      },
+      {
+        Header: t<string>('documents.table.header.actions'),
+        Cell: ({ value, row }: CellProps<TrbAdminTeamHomeRequest>) => {
+          return (
+            <>
+              <UswdsReactLink
+                to={`/trb/${row.original.id}/initial-request-form`}
+                className="margin-right-2"
+              >
+                {t('adminTeamHome.actions.reviewRequest')}
+              </UswdsReactLink>
+              {/* wip assign lead could be a modal popup */}
+              <Button type="button" unstyled>
+                {t('adminTeamHome.actions.assignLead')}
+              </Button>
```

Comment:
This button should be conditionally rendered on the presence/absence of a team lead

Reply:
Ok thanks for finding this